backend/src/main/java/com/eyetrade/backend/model/entity/User.java

```diff
@@ -35,35 +34,36 @@
     @NotEmpty(message = "Please provide your password")
     private String pword;
 
-    @Column
+    @Column(name = "name")
     private String name;
 
-    @Column
+    @Column(name = "surname")
     private String surname;
 
-    @Column
+    @Column(name = "phone")
     private String phone;
 
     @NotNull
-    @Column
+    @Column(name = "role")
     @Enumerated(EnumType.STRING)
     private Role role;
 
-    @Column
+    @Column(name = "iban")
     private String iban;
 
-    @Column
+    @Column(name = "identityNo")
```

Comment:
Because it is a column name, We should use naming type as: "identity_no" instead of camelCase